database/database/seeds/init_engine_db.sql

```diff
@@ -58,6 +58,50 @@ CREATE TABLE public.accels (
 
 ALTER TABLE public.accels OWNER TO postgres;
 
+--
+-- TOC entry ? (class ? OID ?)
+-- Name: trajectories; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public.trajectories (
+    id uuid DEFAULT public.gen_random_uuid() NOT NULL,
+    description character varying(255) NOT NULL,
+    patient_id uuid NOT NULL,
+    x numeric,
+    y numeric,
+    z numeric,
+    created_at timestamp without time zone DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    updated_at timestamp without time zone,
+    deleted_at timestamp without time zone
+);
+
+
+ALTER TABLE public.trajectories OWNER TO postgres;
+
+--
+-- TOC entry ? (class ? OID ?)
+-- Name: tremor; Type: TABLE; Schema: public; Owner: postgres
+--
+
+
+CREATE TABLE public.tremor (
+    id uuid DEFAULT public.gen_random_uuid() NOT NULL,
+    description character varying(255) NOT NULL,
+    patient_id uuid NOT NULL,
+    stationary boolean,
```

Comment:
Stationary, severity, displacement, and frequency should all have comments attached to them

Reply:
Done.